<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferPage</class>
 <widget class="QWidget" name="TransferPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>530</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>转账</string>
   </property>
  </widget>
  <widget class="QToolButton" name="transferRecordBtn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>70</width>
     <height>18</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>转账记录</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>95</y>
     <width>670</width>
     <height>340</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{font: 11px &quot;微软雅黑&quot;;color:rgb(52,37,90);}
#widget{background-color: rgb(255, 255, 255);border-radius:15px;}
</string>
   </property>
   <widget class="QLabel" name="changeAccountLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>来自账户</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sendtoLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>85</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>目标账户地址</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>27</y>
      <width>240</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="amountLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>数量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="amountLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>260</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="placeholderText">
     <string>最多转账金额：1</string>
    </property>
   </widget>
   <widget class="QComboBox" name="assetComboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>55</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QTextEdit" name="memoTextEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>161</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'微软雅黑';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>请输入备注</string>
    </property>
   </widget>
   <widget class="QLabel" name="tipLabel4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>86</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>无效地址</string>
    </property>
   </widget>
   <widget class="QToolButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>255</y>
      <width>96</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>转账</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>币种</string>
    </property>
   </widget>
   <widget class="QComboBox" name="accountComboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>55</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="memoLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>138</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>备注</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tipLabel6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>201</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>长度超过40个字节</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sendtoLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>260</width>
      <height>24</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>请输入对方地址</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_chooseContact">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>95</y>
      <width>12</width>
      <height>12</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tipLabel3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>193</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>余额不足</string>
    </property>
   </widget>
   <zorder>changeAccountLabel</zorder>
   <zorder>sendtoLabel</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>amountLabel</zorder>
   <zorder>amountLineEdit</zorder>
   <zorder>assetComboBox</zorder>
   <zorder>memoTextEdit</zorder>
   <zorder>tipLabel4</zorder>
   <zorder>sendBtn</zorder>
   <zorder>label</zorder>
   <zorder>accountComboBox</zorder>
   <zorder>memoLabel</zorder>
   <zorder>tipLabel6</zorder>
   <zorder>sendtoLineEdit</zorder>
   <zorder>toolButton_chooseContact</zorder>
   <zorder>tipLabel3</zorder>
   <zorder>changeAccountLabel</zorder>
   <zorder>sendtoLabel</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>amountLabel</zorder>
   <zorder>amountLineEdit</zorder>
   <zorder>assetComboBox</zorder>
   <zorder>memoTextEdit</zorder>
   <zorder>tipLabel4</zorder>
   <zorder>sendBtn</zorder>
   <zorder>label</zorder>
   <zorder>accountComboBox</zorder>
   <zorder>memoLabel</zorder>
   <zorder>tipLabel6</zorder>
   <zorder>sendtoLineEdit</zorder>
   <zorder>toolButton_chooseContact</zorder>
   <zorder>tipLabel3</zorder>
  </widget>
  <zorder>widget</zorder>
  <zorder>title_label</zorder>
  <zorder>transferRecordBtn</zorder>
 </widget>
 <tabstops>
  <tabstop>amountLineEdit</tabstop>
  <tabstop>sendBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
